pages/glossary/default-page-language.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Default Page Language
+key: default-page-language
+unambiguous: true
+objective: true
+input_aspects:
+  - DOM tree
+  - Accessibility tree
+  - CSS Styling
+  - Language
+---
+
+The default page language is the _most common language_ used in a [web page][]. If there are no _words_ in the page, the default language is undefined.
+
+The _most common language_ is determined by counting the number of _words_ in the [web page][] that are part of any of the languages in the [language subtag registry][]. The same word can be part of multiple languages. If the number of words is the same, the default language is undefined.
+
+The _words_ used in the following texts should be counted for the _most common language_:
+
+- **page title**: The value of the [document title][] of the [document][] in the [top-level browsing context][].
```

Comment:
I do not understand why browsing context is needed here. "document title" doesn't seem to need it.

Reply:
Because the document title of iframed content is never exposed to AT (at least not as far as I'm aware).

Reply:
Good point.
Should that be an assumption? (for the rule)
"The rule assumes that `iframe` titles are not exposed to AT and thus do not consider them"